### ISSUE TITLE
gii, bootstrap template returns white sceen of death

### ISSUE THREAD

**Issue by gb5256** [cniska/yii-app#35]
Hello out there. 
First of all thanks for providing yii-app.
I have used boilerplate for the last projects, but decided to change to yii-app to use yiistrap. This looks really awesome, well done!

I do have a problem (I guess I installed something wrong).
I installed it as you say in your docs, and appname/web/index.php comes up just fine. So yii is installed ok. I can login as admin etc. 
So far so good.

Now I have changed the gii generator to use the bootstrap one.
I am able to create a model, but the bootstrap-crud shows two problems:
1. If I enter the model name it says that the model does not exist or has errors. But it does work if I enter it with the applciations.modules.Modelname
2. After creating the crud, clicking on TRY OUT gives me the content of the debug sidebar all over the screen, without any css applied, so tons of text. there is no error message at all displayed. I have also tried to disable the debug bar to be able to see any errors, but nothing is displayed. Firebug shows nothing in the console as well.

Any ideas what I have done wrong?
gb5256

**Comment by gb5256** [comment]
Ok, I can answer my first questions my self:
Gii generates a model and puts it into the models folder. But yii-app looks only in the models.ar or models.form folders. So just move the newly created model into ar and it will be detected by gii.

About the second problem: I have added to report error_all to the index and it shows that it needs the ajaxValidation to be a public method, not private as by default. After changing it, it throws another error:

Declaration of BookmarkController::performAjaxValidation() should be compatible with Controller::performAjaxValidation($model, $formId)

So I ended up to remove the whole function as I do not need the Ajax right now.
But this is not solving the problem of course, as it will throw the above problems with every new gii generated crud.
